utils/tte.js

```diff
@@ -21,16 +21,25 @@ const ZERO = 0;
  * @returns {Object} - A tte
  */
 function createTTE(id, team, pdData) {
+  if (!pdData || !pdData.body || !pdData.body.log_entries) return null;
```

Comment:
this could be if (!pdData?.body?.log_entries)

Reply:
Or set entries to pdData?.body?.log_entries and then check if undefined

Reply:
@ConalCosgrove Thanks :) 
I'll try out the optional next time.